charts/posit-chronicle/values.yaml

```
@@ -0,0 +1,85 @@
+image:
+  repository: "935931255537.dkr.ecr.us-east-2.amazonaws.com/chronicle"
```

We should be using the public image now:

```suggestion
  repository: "ghcr.io/rstudio/chronicle"
```

---

Nice! yes let's try this out, i'd forgotten this was public now.